entwatch/ze_ffvii_cosmo_canyon_v5k_fix_override.cfg

```diff
@@ -273,8 +273,8 @@
     }
     "14"
     {
-        "name"              "ZM Cactus"
-        "shortname"         "ZM Cactus"
+        "name"              "ZM Cactuar"
+        "shortname"         "ZM Cactuar"
```

Comment:
Reasoning for this? The map directly calls this item "Zombie cactus" in the chat, as well as the entity names matching.